docs/source/installation/guide.rst

```diff
@@ -145,10 +154,12 @@ Non-GPU environments
 ~~~~~~~~~~~~~~~~~~~~
 
 For users who want to try ChainerMN in a CPU-only environment,
-typically for testing for debugging purpose, ChainerMN can be built
-with the ``--no-nccl`` flag.::
-
-  $ python setup.py install --no-nccl
+typically for testing for debugging purpose, ChainerMN can be used by not installing CuPy.
 
 In this case, the MPI does not have to be CUDA-aware.
 Only ``naive`` communicator works with the CPU mode.
+
+.. note::
+
+   To try current version of ChainerMN in a CPU-only enviroment, you not require ``--no-nccl`` flag,
```

Comment:
Current version of ChainerMN does not need ``--no-nccl`` flag for CPU-only environment at installation any more. It would be just ignored.

Reply:
I will fix it.